addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3031,7 +3031,7 @@ export class OdooEditor extends EventTarget {
             } else if (closestTag === 'TABLE') {
                 this._onTabulationInTable(ev);
             } else if (!ev.shiftKey) {
-                this.execCommand('insert', '\u00A0 \u00A0\u00A0');
+                 this.execCommand('insert', parseHTML('<span class="oe-tabs" contenteditable="false">\u0009</span>\u200B'));
```

Comment:
This line looks weirdly indented on github. Is it a display glitch or is the indentation actually incorrect ?

Reply:
indeed, must be a rebase conflict resolution "error" again. sorry